horovod/common/operations.cc

```diff
@@ -158,6 +167,20 @@ OperationManager* CreateOperationManager(HorovodGlobalState& state) {
 #endif
 #endif
 
+#if COMPILE_WITH_GLOO
+  auto use_gloo = std::getenv("USE_GLOO");
```

Comment:
Instead of `USE_GLOO`, how about `HOROVOD_CPU_OPERATIONS == "gloo"`.  This gives us a couple of advantages:

1. It allows us to add another framework for CPU ops without needing to add further complexity of dealing with the case where multiple USE_X flags are provided.
2. We can distinguish between using Gloo in the data layer vs the control layer.

@alsrgv thoughts?

Reply:
Added code for parsing HOROVOD_CPU_OPERATIONS and HOROCOD_CONTROL_OPERAITINOS flags. If either of the flags is "gloo", we initialized gloo context.

Reply:
I like the idea, but I'd rename `HOROVOD_CONTROL_OPERATIONS` to something else.  Control plane is not an operation from the user perspective.  Maybe just `HOROVOD_CONTROL=GLOO`?